src/tests/components/__snapshots__/Login.test.js.snap

```diff
@@ -0,0 +1,170 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
@kaka, please add snapshots files to your `.gitignore`. You could use this: `**/__snapshots__/**`

Reply:
@MuhanguziDavid , according to the jest documentation, snapshot files are supposed to be included. I had them added to the `.gitignore` and the tests were failing.